routers/web/repo/contributors.go

```diff
@@ -26,22 +24,7 @@ func Contributors(ctx *context.Context) {
 
 	ctx.Data["ContributionTypeText"] = ctx.Tr("repo.contributors.contribution_type." + ctx.Data["ContributionType"].(string))
 
-	if contributor_stats, err := contributors_model.GetContributorStats(ctx, ctx.Repo.Repository, ""); err != nil {
-		ctx.ServerError("GetContributorStats", err)
-		return
-	} else {
-		total_stats, ok := contributor_stats["Total"]
-		if ok {
-			delete(contributor_stats, "Total")
-		}
-		ctx.PageData["repoTotalStats"] = total_stats
-		ctx.PageData["repoContributorsStats"] = contributor_stats
-
-		timeFrom := time.UnixMilli(total_stats.Weeks[0].Week)
-		timeUntil := time.Now()
-		ctx.Data["DateFrom"] = timeFrom.UTC().Format(time.RFC3339)
-		ctx.Data["DateUntil"] = timeUntil.UTC().Format(time.RFC3339)
```

Comment:
I was doing something like this to pass date information to the template. Now that I call api from Vue component, I need to update the template from Vue component. How can I do that?

<img width="1075" alt="image" src="https://github.com/go-gitea/gitea/assets/32161460/f5879336-45d1-4462-ac63-d0aa7e0d7923">

Reply:
You can use `<relative-time>`

Reply:
Can you elaborate on that? What is `<relative-time>` and how it could solve this problem? @yardenshoham

Reply:
See https://github.com/go-gitea/gitea/pull/23988